aerospike-backup-service/templates/deployment.yaml

```
@@ -83,7 +86,12 @@ spec:
           {{- end }}
       volumes:
         - name: aerospike-backup-service-config
+        {{- if .Values.configmap.create -}}
```

```suggestion
        {{ if .Values.configmap.create }}
```
It should solve a helm rendering issue